src/components/dialogs/custom-mui-dialog.tsx

```diff
@@ -102,6 +121,12 @@ function CustomMuiDialog({
                     />
                 </DialogActions>
             </Dialog>
+            <PopupConfirmationDialog
+                message={confirmationMessage}
+                openConfirmationPopup={openConfirmationPopup}
+                setOpenConfirmationPopup={setOpenConfirmationPopup}
+                handlePopupConfirmation={handlePopupConfirmation}
+            />
```

Comment:
maybe enclose it in a condition to not have null case for the msg key

```suggestion
    {confirmationMessage && (
            <PopupConfirmationDialog
                message={confirmationMessage}
                openConfirmationPopup={openConfirmationPopup}
                setOpenConfirmationPopup={setOpenConfirmationPopup}
                handlePopupConfirmation={handlePopupConfirmation}
            />)}
```